lib/src/platform_client.dart

```diff
@@ -400,6 +400,7 @@ class PlatformClient {
           .split(' ')
           .first, // Split the first name and last initial.
       'status': response['serviceStatus'],
+      'agentId': response['agentId'].toString()
```

Comment:
I assumed the `agentId` always a non-null value and this should be fine.

Reply:
`agentId` is Int, trying to make it String for further use.